task/orka-init/0.1/orka-init.yaml

```diff
@@ -0,0 +1,122 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: orka-init
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.16.0"
+    tekton.dev/tags: "orka, macstadium, init, setup"
+    tekton.dev/displayName: "MacStadium Orka Init"
+spec:
+  description: >-
+    With this set of Tasks, you can use your Orka environment
+    to run macOS builds and macOS-related testing from your Tekton pipelines.
+
+    This Task creates a VM template with the specified configuration.
+    All operations in this Task are performed against an Orka environment.
+  params:
+    - name: base-image
+      type: string
+      description: The Orka base image to use for the VM config.
+    - name: cpu-count
+      type: string
+      description: |
+        The number of CPU cores to dedicate for the VM. Must be 3, 4, 6, 8, 12, or 24.
+      default: "3"
+    - name: vcpu-count
+      type: string
+      description: |
+        The number of vCPUs for the VM. Must equal the number of CPUs, when CPU is less than or equal to 3.
+        Otherwise, must equal half of or exactly the number of CPUs specified.
+      default: "3"
+    - name: vnc-console
+      type: string
+      description: Enables or disables VNC for the VM.
+      default: "true"
+    - name: orka-creds-secret
+      type: string
+      description: The name of the secret holding your Orka credentials.
+      default: orka-creds
+    - name: orka-creds-email-key
+      type: string
+      description: |
+        The name of the key in the Orka user credentials secret for the email address associated with the Orka user.
+      default: email
+    - name: orka-creds-password-key
+      type: string
+      description: |
+        The name of the key in the Orka credentials secret for the password associated with the Orka user.
+      default: password
+    - name: orka-token-secret
+      type: string
+      description: |
+        The name of the secret holding the authentication token used to access the Orka API.
+      default: orka-token
+    - name: orka-token-secret-key
+      type: string
+      description: |
+        The name of the key in the Orka token secret, which holds the authentication token.
+      default: token
+    - name: orka-vm-name-config
+      type: string
+      description: |
+        The name of the config map, which stores the name of the generated VM configuration.
+      default: orka-vm-name
+    - name: orka-vm-name-config-key
+      type: string
+      description: |
+        The name of the key in the VM name config map, which stores the name of the generated VM configuration.
+      default: vm-name
+  results:
+    - name: vm-name
```

Comment:
are we storing any results?

Reply:
Good catch - this is left over from a previous implementation and is no longer used.